Feature/modify base manifest sample (#5)
st-tech/gatling-commander: docs/user-guide.md

The `base_manifest.yaml` describes the common values for each load test in the Kubernetes manifest of the Gatling Resource.

Fields marked `<config.yaml overrides this field>` in `base_manifest.yaml` are set to different values for each load test. The value of this field will be replaced by the value in `config.yaml` respectively when Gatling Commander runs. Therefore, setting values to fields marked `<config.yaml overrides this field>` in `base_manifest.yaml` is not necessary.

> \* Gatling Commander once loads `base_manifest.yaml` value to Gatling struct object before replaced by `config.yaml`. So the type of `base_manifest.yaml` field vlaue must be matched to Gatling struct field one. If type not match, like following error will be occured.

Reviewer: @okmtz 
【IMO】
Corrected sentences.
```suggestion
\* Gatling Commander loads `base_manifest.yaml` value to Gatling struct object before it replaces the value by `config.yaml`. So the type of `base_manifest.yaml` field value must be matched to Gatling struct field one. If type not match, an error like following will occur.
```

Author: @kayamin 
commit suggestion and rebase with my old commit !
thank you !

> \* Gatling Commander once loads `base_manifest.yaml` value to Gatling struct object before it replaces the value by `config.yaml`. So the type of `base_manifest.yaml` field value must be matched to Gatling struct field one. If type not match, an error like following error will be occured.

Reviewer: 【nits】
```suggestion
\* Gatling Commander once loads `base_manifest.yaml` value to Gatling struct object before it replaces the value by `config.yaml`. So the type of `base_manifest.yaml` field value must be matched to Gatling struct field one. If type not match, an error like following occur.
```
